news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,291 @@
+
+Unfortunately, I don't have plans to go for full completion. I got exhausted enough grinding for 100,000 SS, and going for completionist means that I would need to play way harder maps in order to accomplish to it (and not even get SS!!!), and I don't feel I have hands/arms to grind so much again (especially because I get tired quite fast)... so I feel it's more worth it if I focus in other things now. But I really really admire completionist players, and also the ones that want to be, no matter what game mode! It's a very long journey, and I know it's not easy.
+
+BUT I have plans in going for osu!catch (only exclusive maps) completionist. It's much less maps, and feels more realistic to me. Not too soon, though. That's something for later!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Following up on the previous question, what are your thoughts on the current state of completionist badge?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+I think that the badges are really cool, it needs to be updated more frequently, though... but PERSONALLY, I feel titles are more cool in completionist cases. It's very very unique thing to do, and it just keeps getting harder and harder to complete the game, since there are maps getting ranked everyday!
+But related to that, same as what to happens in [this page](/wiki/People/Completionists), I feel it would be cool to have some page in the wiki that register players that grind other things and reach specific marks too, and when. As for example SS and Ranked Score, which are some of other things that many usually grind nowadays. Who already reached 50k SS in osu!catch and WHEN did they reach it? and in osu!? Taiko? osu!mania? I feel those things are important not only to keep history registered, but to all players that grinds other things also get recognized by their efforts too, which is usually understimated and underappreciated.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+As an osu!catch player, what are your thoughts on Lazer?
```

Comment:
```suggestion
As an osu!catch player, what are your thoughts on osu!(lazer)?
```

Reply:
Please refer to https://github.com/ppy/osu-wiki/pull/12103#discussion_r1761460587

Reply:
@Walavouchey don't we already use `osu!(lazer)` all the time? if not, re-opening regardless to suggest a lowercase lazer for consistency with xooty's interview

```suggestion
As an osu!catch player, what are your thoughts on osu!(lazer)?
```
or
```suggestion
As an osu!catch player, what are your thoughts on lazer?
```

Reply:
also whatever the consensus for this should apply to other instances of lazer here and onwards.

Reply:
it should always be osu!(lazer), the exception being the interviewee's answers since those should only be changed if there are grammar issues or slightly reworded to get the point across better while not harming the original statement

Reply:
osu!(lazer) predominantly, but "lazer" is fine in this context where it's specifically referring to the release stream and by no means the game itself (this isn't the case in many other posts)